packages/loader/src/gltf/parser/GLTFParserContext.ts

```diff
@@ -34,7 +43,10 @@ export class GLTFParserContext {
   contentRestorer: GLTFContentRestorer;
 
   /** @internal */
-  _buffers: ArrayBuffer[] | Promise<ArrayBuffer[]>;
+  _buffers?: ArrayBuffer[];
+
+  /** @internal */
+  _cache = new Map<string, any>();
```

Comment:
What cache? name and type should more clear

Reply:
done